src/algebra/ordered_monoid.lean

```diff
@@ -975,3 +977,16 @@ instance [linear_ordered_comm_monoid α] : linear_ordered_add_comm_monoid (addit
   ..additive.ordered_add_comm_monoid }
 
 end type_tags
+
+/-- The order embedding sending `b` to `a * b`, for some fixed `a`. -/
+@[to_additive "The order embedding sending `b` to `a + b`, for some fixed `a`.", simps]
+def order_embedding.mul_left
+  {α : Type*} [has_mul α] [linear_order α] [covariant_class α α (*) (<)] (m : α) : α ↪o α :=
+order_embedding.of_strict_mono (λ n, m * n) (λ a b w, mul_lt_mul_left' w m)
```

Comment:
I'm slightly surprised this needs `linear_order` when [`order_iso.mul_left`](https://leanprover-community.github.io/mathlib_docs/algebra/ordered_group.html#order_iso.mul_left) doesn't.